src/styles/_header.scss

```diff
@@ -25,6 +25,7 @@
     color: #060b35;
     list-style: none;
     line-height: 60px;
+    transition: 400ms ease-in-out;
```

Comment:
Always add transition property. By default, transition property is all, and it's hard for the browser to control all properties